lib/zip/file.rb

```diff
@@ -51,21 +51,31 @@ class File < CentralDirectory
     DATA_BUFFER_SIZE     = 8192
     IO_METHODS           = [:tell, :seek, :read, :close]
 
+    DEFAULT_OPTIONS = {
+      restore_ownership:   false,
+      restore_permissions: false,
+      restore_times:       true
```

Comment:
While the default value for `restore_times` was notionally `true` in the original code, it didn't actually do anything (as you discovered), so it effectively defaulted to `false`. For compatibility, perhaps this should default `false` for now?

Reply:
I think that restoring the original times probably should be the eventual default - I've seen use-cases where the whole point of zipping a directory up and transferring it was to preserve the timestamps.

But I take your point that it might be better to default to false for now. I'll change that.